engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -34,6 +38,8 @@ class EngineJobExecutionActor(jobKey: BackendJobDescriptorKey,
   val jobTag = s"${workflowId.shortString}:${jobKey.call.fullyQualifiedName}:${jobKey.index.fromIndex}:${jobKey.attempt}"
   val tag = s"EJEA_$jobTag"
 
+  private var callCachingActorProps: Props = _
```

Comment:
Yes I think that's right, but nothing should try to access it if it isn't set because cache reading would be off.